src/components/views/rooms/MessageComposer.js

```diff
@@ -331,7 +334,17 @@ export default class MessageComposer extends React.Component {
     }
 
     _onVoiceStoreUpdate = () => {
-        this.setState({haveRecording: !!VoiceRecordingStore.instance.activeRecording});
+        const recording = VoiceRecordingStore.instance.activeRecording;
+        this.setState({haveRecording: !!recording});
+        if (recording) {
+            // We show a little head's up that the recording is about to automatically end soon. The 3s
```

Comment:
No apostrophe, see https://idioms.thefreedictionary.com/heads+up and such.

Reply:
I blame Canada